components/CreateEvent.js

```diff
@@ -137,7 +137,7 @@ class CreateEvent extends React.Component {
           </div>
         </Body>
 
-        <Footer />
+        <Footer LoggedInUser={LoggedInUser} />
```

Comment:
Passing this as a prop each time we use `Footer` is tedious and easy to miss when implementing new pages. We should rather use the `useUser` hook: https://github.com/opencollective/opencollective-frontend/blob/8bbb7cdc1c12770c4e73173147ebc4224d5c95bf/components/UserProvider.js#L162